### PR TITLE
Migrate default templates to render functions

### PR DESCRIPTION
**Pull request by cristinecula**
Re https://github.com/Neovici/cosmoz-frontend/issues/3282

### CODE REVIEW

**Comment by cristinecula** [comment]
@megheaiulian please review, but don't merge yet, I need to add some more tests.

**Comment by cristinecula** [comment]
I have updated the tests to cover most of the diff.

File: cosmoz-omnitable-column-autocomplete.js [hunk 1]
```diff
@@ -72,6 +40,42 @@ class OmnitableColumnAutocomplete extends listColumnMixin(columnMixin(PolymerEle
...
+			no-label-float
+			type="text"
+			@change=${ onChange }
+			.value=${ column.getString(item, column.valuePath) }
+		></paper-input>`;
+	}
+
+	renderHeader(column) {
+		const spinner = column.loading
+			? html`<paper-spinner-lite style="width: 20px; height: 20px;" suffix slot="suffix" active></paper-spinner-lite>`
```
Comment (megheaiulian) [inline cosmoz-omnitable-column-autocomplete.js:63]: We should next use the lightweight spinner that we have  in cosmoz-autocomplete. Maybe add a issue about it ?

**Review by megheaiulian — APPROVED** [review]
Looks very good!

**Pull request merged**